network/peer/peer.go

```diff
@@ -972,6 +973,9 @@ func (p *peer) handleVersion(msg *p2p.Version) {
 		return
 	}
 
+	// add clock skew to metric
+	p.Metrics.ClocksSkew.Add(clockDifference)
```

Comment:
I feel like this might make more sense to measure before the filter due to the `MaxClockDifference`. I guess the question is if we are trying to measure the clock skew of our peers - or if we are trying to measure the clock skew of people trying to join the network.

Reply:
So you prefer to measure the clock skew of people trying to join the network 👍

Reply:
done